docs/learn/guides/messaging/messaging-with-sqs/README.md

```diff
@@ -0,0 +1,113 @@
+# Messaging with SQS
+
+This guide demonstrates how to use Amazon SQS Connector to notify alerts.
+
+The high level sections of this guide are as follows:
+
+- [What you'll build](#What-youll-build)
+- [Prerequisites](#Prerequisites)
+- [Implementation](#Implementation)
+  - [Creating the project structure](#Creating-the-project-structure)
+  - [Developing the scenario](#Developing-the-scenario)
+- [Deployment](#Deployment)
+  - [Deploying locally](#Deploying-locally)
+  - [Deploying on Docker](#Deploying-on-Docker)
+
+## What you'll build
+
+Let's consider a scenario where a fire alarm send fire alerts to a Amazon SQS queue. As the message duplication is not an issue, an Amazon SQS Standard Queue is used as the queue. A fire alarm listener is polling the messages in the SQS queue. When fire alerts are available, it will consume the messages from the queue and remove the messages from the queue. Consumed messages are showed in the console to the User. 
+
+Here, the fire alarm is sending fire alerts periodically from a Ballerina worker where listener polls in another worker. Both sent messages and received messages are printed in the console. 
+
+As there can be multiple alert messages available in the queue, the listener is configured to consume more than one message at a time.
+
+The following diagram illustrates the scenario:
+
+![Message flow diagram image](resources/sqs-alert.png)
```

Comment:
Shall we have the image above an explanation of the image? The idea is that a user first sees the image and then looks below if more details are required.

Reply:
Moved the image.